packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { CardVariables } from './cardVariables';
+import { CardStylesProps } from '../../../../components/Card/Card';
+import { pxToRem } from '../../../../utils';
+
+const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      display: 'flex',
+      flexDirection: 'column',
+      position: 'relative',
+      padding: v.cardPadding,
+      ...(p.horizontal && { flexDirection: 'row' }),
+      ...(p.compact && { padding: 0 }),
+      ...(p.centered && { alignItems: 'center' }),
+
+      // TODO: update with latest design spec
+      width: pxToRem(300),
+      border: '1px solid black',
```

Comment:
Let's move this too `variables` at least. We also have some borders related stuff under `siteVariables`:

![image](https://user-images.githubusercontent.com/14183168/77151142-a3dc0c00-6a95-11ea-8c84-4da28fa93c6a.png)